src/collections/programs/lfx-2023/lfx-2023.mdx

```diff
@@ -113,15 +113,73 @@ A use-case that fits very well is be the ability to use it to run performance te
+
+**Expected Outcome**: a powerful real-time multi-user collaboration experience.
+- Recommended Skills: Golang, Open Policy Agent, WASM
+- Mentor(s): <Link to="/community/members/lee-calcote">Lee Calcote</Link>, <Link to = "/community/members/abhishek-kumar">Abhishek Kumar</Link>
+- Upstream Issue (URL): https://github.com/meshery/meshery/issues/7019
+
+#### Adopt OCI as the packaging and distribution format for Meshery MeshModels
+
+**Description**: Meshery MeshModels represent a schema-based description of cloud native infratructure. MeshModels need to be portable between Meshery deployments as well as easily versionable in external repositories. 
+
+**Expected Outcome**: 
+1. Meshery clients (mesheryctl and Meshery UI) should be able to import/export MeshModels as OCI images.
+2. Meshery clients (mesheryctl and Meshery UI) should be able to push/pull from OCI-compatible registries.
+3. Stretch Goal: OCI image signing; Verify the authenticity of MeshModels using cosign (https://github.com/sigstore/cosign).
+4. Target registries: Meshery Catalog (https://meshery.io/catalog), Artifact Hub.
+
+
+- Recommended Skills: Reactjs, Golang, GraphQL
+- Mentor(s): <Link to="/community/members/lee-calcote">Lee Calcote</Link>
+- Upstream Issue (URL): https://github.com/meshery/meshery/issues/6447
+
+### Service Mesh Performance
+<br />
+
+#### Service Mesh Performance IDE Plugin
+
+**Description**: The objective of this project is to develop IDE plugins that can enhance the developer experience while working with Service Mesh Performance Performance Profiles. The proposed plugins will leverage technologies such as golang and cuelang to provide features such as syntax highlighting, auto-completion, validation, and rendering previews for Service Mesh Performance profile and model definitions. 
+
+**Expected Outcome**: 
+1. Release VS Code Extension
+2. Syntax Highlighting and Auto-completion: The plugin can fetch SMP Model definitions such as cloud-native components and their relationships. This information can be used to provide syntax highlighting and auto-completion for these definitions in the JSON files, making it easier for developers to write error-free code.
+3. Validation and Reference: For Meshery MeshModel definitions such as cloud-native components and their relationships, the plugin can use the CUE language to provide validation for the CUE input and preview the rendering result. The plugin can also fetch the SMP Model schemas and display them in the IDE for reference.
+
+
+- Recommended Skills: Cuelang
+- Mentor(s): <Link to="/community/members/lee-calcote">Lee Calcote</Link>, <Link to = "/community/members/xin-huang">Xin Huang</Link>
+- Upstream Issue (URL): https://github.com/service-mesh-performance/service-mesh-performance/issues/379
+
+
 ## Additional information
 <br />
 
```

Comment:
in [Additional information](https://github.com/layer5io/layer5/blob/master/src/collections/programs/lfx-2023/lfx-2023.mdx#additional-information): 
<p>
Our mentors have managed teams of technical writers working on documenting enterprise-grade software at large technology companies (Cisco, Seagate, SolarWinds). During the span of time, <code>he has worked</code> with technical writers in DITA and post-DITA environments (from Word to FrameMaker, structured writing, online help, various CMSes, git). Our mentors have worked with technical writers on documentation strategy, creating document sets, covering the full spectrum of reader personas.
</p>
will you replace the <code>he has worked with </code> to <code>they have worked with</code> @aku1310 ?